CRT.c

```diff
@@ -203,6 +203,8 @@ static int CRT_colorSchemes[LAST_COLORSCHEME][LAST_COLORELEMENT] = {
       [PRESSURE_STALL_THREEHUNDRED] = ColorPair(Cyan, Black),
       [PRESSURE_STALL_SIXTY] = A_BOLD | ColorPair(Cyan, Black),
       [PRESSURE_STALL_TEN] = A_BOLD | ColorPair(White, Black),
+      [FILE_DESCRIPTOR_USED] = ColorPair(Green, Black),
+      [FILE_DESCRIPTOR_MAX] = ColorPair(Red, Black),
```

Comment:
Do we want to use red here, or a less alerting color?

Reply:
I don't mind. Any suggestions?

Reply:
Maybe `Magenta` or `Blue`?

Reply:
Okay, I think bold blue on black should do then …